packages/addons/service/prometheus-node-exporter/source/bin/prometheus-node-exporter.start

```diff
@@ -0,0 +1,205 @@
+#!/bin/sh
+
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (C) 2009-2014 Stephan Raue (stephan@openelec.tv)
+# Copyright (C) 2016-present Team LibreELEC (https://libreelec.tv)
```

Comment:
```suggestion
# SPDX-License-Identifier: GPL-2.0
# Copyright (C) 2021-present Team LibreELEC (https://libreelec.tv)
```

Reply:
I guess this package didn't exist at OpeneELEC before ?

Reply:
Yup , many cut paste from others addons , sorry :)
Fixed